Biobanks metadata with MIABIS structure

svituz opened the pull request:
This PR changes the metadata schema for biobanks in order to be compliant with the [MIABIS](https://github.com/BBMRI-ERIC/miabis) schema

NB: currently it is missing the PNG with the representation of the schema

svituz committed:
feat: change of biobank resource to map MIABIS concepts

svituz committed:
feat: change of biobank resource to map MIABIS concepts

svituz committed:
feat: changes example for biobank

svituz committed:
feat: changes example for biobank

svituz committed:
adds: biobank ttl example

svituz committed:
feat: changes description of biobanks

svituz committed:
--amend

henrietteharmse reviewed (commented):
Here are some initial feedback for a discussion.

**Valid points**
1. The `dcat:Resource` in the EJP RD model is missing the `dct:indentifier` property that is very clearly part of DCAT2 and even if it was not part of DCAT2 it should have been added to realize the Findable aspect of data and metadata. This seems to be an oversight of the EJP RD model.
2. Basing biobank related concepts on OBIB is a sound idea that we may want to encourage.
3. The EJP RD model does not include `dcat:contactPoint` on `dcat:Resource`. Why? This would seem to make sense as both DCAT2 and FDP have it. 

**To discuss**
1. The intent of the EJP RD meta-model is to only deal with meta-data. It therefore would seem that collections should not form part of meta-data as these in fact represent record data of biobanks. It would be similar to adding patient and diagnosis information for patient registries to the meta-data model. The fact that a collection seem to be a record-level element seems to be supported by the definition of OBIB_0000645  "A object aggregate that consists of specimens, participates in a study and is part of a biobank." Here is diagram representing collection data:
![image](https://github.com/ejp-rd-vp/resource-metadata-schema/assets/32327979/b7716a17-3fe6-468d-bea6-086d40bf2f6c)

2. Should the EJPRD model include ` fdp-o:metadataIdentifier` information? This seems very specific to FDP and not general the EJPRD meta-data model. Also, it seems that ` fdp-o:metadataIdentifier` is deprecated based on [https://w3id.org/fdp/fdp-o#](See https://w3id.org/fdp/fdp-o#). However, when looking at [https://specs.fairdatapoint.org/](https://specs.fairdatapoint.org/) 
![image](https://github.com/ejp-rd-vp/resource-metadata-schema/assets/32327979/860119c5-3cfd-4d65-a42f-febb05aa9966) it seems to still in use.
4. 


**Possible error?**
1. `dct:isPartOf :catalog` but `:catalog` is not defined.

svituz committed:
feat/miabis_biobanks

svituz committed:
fix: fixes obib term for biobank organization in biobank example

svituz commented:
I changed the mapping of the concepts of biobanks and collections. 
I do agree that the previous mapping of the concepts defined by OBIB and MIABIS didn't completely fit.

Now the MIABIS Collection is mapped to `obib:biobank` and the MIABIS Biobank is mapped to `obib:biobank_organization`. The two entities are linked by the relations `:biobank obib:owns :collection` and `:collection obib:is_owned_by :biobank`. The biobank organization has now a `dcat:publisher` -> `foaf:Organization`, similarly to the Patient Registry and to the old version of the biobank model.
This new version should be more similar to the old EJP model, with the difference that it uses OBIB concepts and relations. Indeed the old `ejp:Biobank` is now `obib:biobank` while the `foaf:Organization` is now denoted more specifically using the `obib:biobank_organization` term.

This mapping should also clarify the doubt about collections being records of the biobank: the collection metadata describes the samples while the MIABIS biobank contains only organizational data. For example, the metadata about the diseases treated is registered in the collection, from a MIABIS point of view. 

Other minor changes: I removed the FDP specific attributes from the examples and the `dcat:isPartOf :catalog`

svituz committed:
feat: changes the relation between collection and biobank, changes the biobankShape to be a foaf:Organization and introdue the legal entity for biobank

svituz committed:
feat: changes biobank to be more similar to Dataset and compatible with both MIABIS and previous EJP model

svituz commented:
I changed again the mapping. Now it is really similar to the EJP previous model.
The main changes have been restricted to:

- The old `ejp:Biobank` entity has been replaced by `dcat:Dataset` and `obo:OBIB_0000616` (obib:biobank) to be compatible with MIABIS
- The old `foaf:Organization` representing the Organization that owns the Biobank has been enhanced as follows:
   - It has been also assigned the `obo:OBIB_0000623` (obib:biobank_organization) to specify it is a Biobank Organization as defined by MIABIS
   - the `obo:OBIB_0000732` (obib:owns) relates the biobank to the collectionShape
   - the `obo:RO_0000053` (obo:has_charasteristic) predicate has been added. This generic predicate from obo is used to link the biobank to an object of type `obo:OMRSE_00000038` (obo:legal_entity) to link it to the Legal Entity (e.g. a university) that represents the Organization

svituz commented on data/example-rdf/turtle/biobank.ttl:
```diff
@@ -1,26 +1,52 @@
+  dct:title "Biobank of Rare Diseases";
+  dct:alternative "BRD";
+  dct:description "Biobank with collections of biopsies of muscular diseases";
+  dcat:landingPage <https://brd.org>;
```
should be foaf:homepage

svituz commented on data/example-rdf/turtle/biobank.ttl:
```diff
@@ -1,26 +1,52 @@
+  a obo:OBIB_0000623, foaf:Organization;
+  dct:identifier "biobank-id";
+  dct:title "Biobank of Rare Diseases";
+  dct:alternative "BRD";
```
not in the current model

svituz commented on data/example-rdf/turtle/biobank.ttl:
```diff
@@ -1,26 +1,52 @@
+  dct:identifier "collection-id";
+  dct:title "Muscle Tissue Collection";
+  dct:description "Example collection of samples";
+  dct:alternative "MTC";
```
not in the current model

Orphanet replied:
I think, there is a typo here:
dcat:Resouce;

should be dcat:Resource;

svituz replied:
I changed it in the new commit

svituz committed:
feat: removes dct:alternative and other minor changes
